packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md

```diff
@@ -0,0 +1,64 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Announcing Lit 3.0 Pre-releases"
+summary: "Get an early look at the upcoming Lit 3.0 release."
+date: 2023-05-15
+author:
+  - justin-fagnani
+---
+
+# Announcing Lit 3.0 Pre-releases
+
+The Lit team is excited to announce the first pre-releases of Lit 3.0!
+
+Lit 3.0 is our first major version since [introducing the unified Lit project and Lit 2.0](https://lit.dev/blog/2021-04-21-lit-2.0-meet-lit-all-over-again/) two years ago. We really value stability and backwards compatibility for our community, so we've focused on adding new features with minor versions of the core libraries and new labs packages, and not making any breaking changes.
+
+But the time is right for just a few breaking changes that will improve development velocity and testing stability on the core Lit project.
+
+## Changes
+
+Lit 3.0 adds no new features, because new features are generally not breaking changes and can be added in minor versions, according to semver. The Lit 3.0 release is an opportunity to make a few breaking changes that trim out some technical debt to unlock new features we have scheduled for our 3.x release series.
+
+The Lit 3.0 changes are mostly in browser support, removing deprecated APIs, and how publish packages. If you run Lit 2.x with no deprecation warnings, this should be a seamless upgrade!
+
+Here are the biggest things Lit 3.0 changes:
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated during the Lit 1.x release timeframe have been removed.
+- SSR hydration support modules were moved to the `@lit-labs/ssr-client` package.
+
+Detailed change logs can be found [on GitHub](https://github.com/lit/lit/releases?q=%22-pre.0%22&expanded=true).
+
+## Trying the Pre-releases
+
+We would love your help testing the new versions, to ensure a smooth upgrade process with the final releases. We're especially interested in making sure the new language version works with your toolchains. We expect some users may need to upgrade their tooling to the latest versions.
+
+You can try the pre-releases out by updating your package.json file to include the following:
+
+```json
+  "lit": "^3.0.0-pre.0"
+  "lit-html": "^3.0.0-pre.0"
+  "lit-element": "^4.0.0-pre.0"
+  "@lit/reactive-element": "^2.0.0-pre.0"
+```
+
+You can also use the `pre` npm tag, like `npm i lit@pre`.
+
+All other packages, like labs packages, have pre-release versions too that depend on the prerelease core libraries. If you depend on those you'll have to update them too. If you're more daring you can use npm overrides to select the pre-releases even for dependencies. This should work for most dependencies.
```

Comment:
I see more pre-release than prerelease so I think we'll go with the hyphenated version consistently?
```suggestion
All other packages, like labs packages, have pre-release versions too that depend on the pre-release core libraries. If you depend on those you'll have to update them too. If you're more daring you can use npm overrides to select the pre-releases even for dependencies. This should work for most dependencies.
```